fastlane/Fastfile

```diff
@@ -0,0 +1,83 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+default_platform(:ios)
+
+platform :ios do
+
+  #### Pod ####
+
+  desc "Lint MatrixSDK podspec"
+  lane :pods_lint do  	
+  	pod_lint_MatrixSDK
+  	pod_lint_SwiftMatrixSDK
+  end
+
+  desc "Lint MatrixSDK podspec"
+  lane :pod_lint_MatrixSDK do  	
+  	custom_pod_lib_lint(podspec: "../MatrixSDK.podspec", parameters: ['--use-libraries', '--allow-warnings'])
+  end
+
+  desc "Push MatrixSDK pod"
+    lane :push_pod_MatrixSDK do  	
```

Comment:
Could `pod_lint_MatrixSDK` & `push_pod_MatrixSDK` and others follow the same naming convention? verb first or `pod` first?